ds_autotuning_prototype/hf_trainer_api/image_classification/autotune_config.yaml

```diff
@@ -0,0 +1,41 @@
+name: vision transformer deepspeed stage 1
+debug: false
+environment:
+    environment_variables:
+        - NCCL_DEBUG=INFO
+        # You may need to modify this to match your network configuration.
+        - NCCL_SOCKET_IFNAME=ens,eth,ib
+    image:
+      gpu: determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-deepspeed-0.7.0-gpu-9119094
+resources:
+  slots_per_trial: 2
+searcher:
+  name: custom
+  metric: throughput
+  smaller_is_better: false
+hyperparameters:
+  training_arguments:
+    learning_rate: 1e-5
+entrypoint: >-
+  python -m determined.launch.deepspeed
+  python image_classification.py 
+  --dataset_name beans  
+  --output_dir ./beans_outputs/  
+  --remove_unused_columns False 
+  --do_train  
+  --do_eval 
+  --max_steps 100
+  --per_device_train_batch_size 8 
+  --per_device_eval_batch_size 8
+  --logging_strategy steps 
+  --logging_steps 10 
+  --evaluation_strategy steps
+  --eval_steps 10
+  --save_total_limit 3 
+  --seed 1337
+  --save_strategy steps
+  --save_steps 20
+  --deepspeed ds_configs/ds_config_stage_1.json
+  --cache_dir ./garrettgoon/cache_dir
```

Comment:
This is made on the remote searcher right? If it helps, why would we want to remove it in prod?

Reply:
this whole file shouldn't be there, since we don't need a separate autotuning config anymore. just deleted it.

Not sure what you were asking re: removing something though?

Reply:
the `cache_dir` parameter. The comment below implies we don't want this parameter long term

Reply:
oh yeah, here I am specifying some external storage which I mount to my containers on Grenoble. this makes it so you don't have to re-download data sets or model weights for every new trial; they are cached instead.

the path specified here is just very specific to this workflow, so it needs to be removed.